sw/device/lib/crypto/impl/rsa/rsa_padding.c

```diff
@@ -12,6 +12,17 @@
 #include "sw/device/lib/crypto/impl/sha2/sha512.h"
 #include "sw/device/lib/crypto/include/hash.h"
 
+enum {
```

Comment:
The SHA-2 variants of these constants are defined in `sw/device/lib/crypto/impl/sha2/`, but there is currently no equivalent for SHA-3. For now, I define the constants here, but a later PR should create a `sw/device/lib/crypto/impl/sha3/` directory and declare the constants there.

Reply:
Actually, since SHA3 is a hardware implementation, I'd just throw these into the [KMAC driver header](https://github.com/lowRISC/opentitan/blob/8faf04697a14d8d04ab2c126a5aa8d99976e56a3/sw/device/lib/crypto/drivers/kmac.h#L19). Then the KMAC driver itself could also use them to replace some magic numbers, e.g.:
https://github.com/lowRISC/opentitan/blob/8faf04697a14d8d04ab2c126a5aa8d99976e56a3/sw/device/lib/crypto/drivers/kmac.c#L717

Reply:
Done!